frontend/src/pages/Alerts/AlertGraph/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { Box, DateRangePreset } from '@highlight-run/ui/components'
+import React from 'react'
+import { ReferenceArea, ReferenceLine } from 'recharts'
+
+import { MetricAggregator, ProductType } from '@/graph/generated/schemas'
+import { useProjectId } from '@/hooks/useProjectId'
+import Graph, {
+	getViewConfig,
+	SetTimeRange,
+} from '@/pages/Graphing/components/Graph'
+
+import * as style from './styles.css'
+
+const EditorBackground = () => {
+	return (
+		<svg width="100%" height="100%">
+			<defs>
+				<pattern
+					id="polka-dots"
+					x="0"
+					y="0"
+					width="14"
+					height="14"
+					patternUnits="userSpaceOnUse"
+				>
+					<circle fill="#e4e2e4" cx="7" cy="7" r="1" />
+				</pattern>
+			</defs>
+
+			<rect
+				x="0"
+				y="0"
+				width="100%"
+				height="100%"
+				fill="url(#polka-dots)"
+			/>
+		</svg>
+	)
+}
+
+type Props = {
+	alertName: string
+	query: string
+	productType: ProductType
+	functionColumn: string
+	functionType: MetricAggregator
+	groupByKey?: string
+	thresholdWindow: number
+	thresholdValue: number
+	belowThreshold: boolean
+	startDate: Date
+	endDate: Date
+	selectedPreset?: DateRangePreset
+	updateSearchTime: SetTimeRange
+}
+
+export const AlertGraph: React.FC<Props> = ({
+	alertName,
+	query,
+	productType,
+	functionColumn,
+	functionType,
+	groupByKey,
+	thresholdWindow,
+	thresholdValue,
+	belowThreshold,
+	startDate,
+	endDate,
+	selectedPreset,
+	updateSearchTime,
+}) => {
+	const { projectId } = useProjectId()
+	const sessionsProduct = productType === ProductType.Sessions
+
+	const viewConfig = sessionsProduct
+		? getViewConfig('Bar chart', 'Stacked', 'Zero')
+		: getViewConfig('Line chart', 'Line', 'Zero')
```

Comment:
if a custom has >1000 sessions per time interval of the session alert, could there be any issues with rendering that many individual bars / with the `group by LIMIT` on the backend? just wanted to check that case will work / that we shouldn't render 1 bar for all the number of sessions